Fix Media events `ended` missing info (#34761)
mdn/content: files/en-us/web/api/htmlmediaelement/ended_event/index.md

This event is not cancelable and does not bubble.

> > **Note:** The `ended` event will not be fired if the `loop` attribute is set to true on the HTMLMediaElement.

Reviewer: ```suggestion
> **Note:** The `ended` event doesn't fire if the [`loop`](/en-US/docs/Web/API/HTMLMediaElement/loop) property is `true` and [`playbackRate`](/en-US/docs/Web/API/HTMLMediaElement/playbackRate) is non-negative.
```
